### ISSUE TITLE
Date in repeater not saved 

### ISSUE THREAD

**Issue by parijke** [bolt/bolt#4742]
Related with #4706

The error is gone. But there are two more (related?) issues:

- The dates aren't saved
- The popup calendar doesn't work

**Comment by rossriley** [comment]
Any chance you could paste the contenttypes config you are using for the repeater field. I'm seeing everything working OK here but it would be good to get an exact replica to try and reproduce.

**Comment by parijke** [comment]
Any change of course:
```
entries:
    name: Entries
    singular_name: Entry
    fields:
        title:
            type: text
            class: large
            group: content
        slug:
            type: slug
            uses: title
        teaser:
            type: html
            height: 150px
        body:
            type: html
            height: 300px
        image:
            type: image
            group: media
        video:
            type: video
        repeater:
            type: repeater
#            group: repeater
            limit: 0
#            prefix: "<p>This allows you to create multiple sets of fields. Use the add button at the bottom to create a new empty set.</p>"
            fields:
                date:
                    type: date
                    variant: inline
```

**Comment by rossriley** [comment]
This should be fixed in #4756
